doc/tutorials/platformer/step_3.md

```diff
@@ -422,7 +422,7 @@ with a `passive` `CollisionType`.  Collisions will be explained more in a later
 #### Display the Platform
 
 In our `loadGameSegments` method from earlier, we will need to add the call to add our block.  We
-will need to define `gridPosition` and `xOffset` to be passed in.  `gridPostion` will be a
+will need to define `gridPosition` and `xOffset` to be passed in.  `gridPosition` will be a
```

Comment:
How come spell didn't catch this? Should we add it to the `flagWords` in cspell.json?

Reply:
because we have it set to ignore anything within backticks via regex on config. I am working on improving that to be more strict